packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware/index.ts

```diff
@@ -64,6 +64,17 @@ const analyticsMiddleware = () => (store) => (next) => (action) => {
 
         const pageName: PageName = action.payload.location.pathname
 
+        try {
+          // @ts-ignore
+          if (window && window.gtag) {
+            // @ts-ignore
+            window.gtag('set', 'page_path', pageName)
+            // @ts-ignore
```

Comment:
add `gtag` to `src/typings/window.d.ts` so you don't have to use `@ts-ignore`'s

Reply:
there is dedicated typing https://github.com/DefinitelyTyped/DefinitelyTyped/tree/4d4da2106dcd54af4a0ddeb5e0360f322fd8a5bf/types/gtag.js

Reply:
Even from beyond the grave @schnogz still got our backs. <3